pkg/runners/suite_runner.go

```diff
@@ -42,8 +42,9 @@ func (s *SuiteRunner) Run(ctx context.Context, envs []string, writer *bufio.Writ
 		exechelper.WithStdout(writer),
 		exechelper.WithStderr(writer),
 		exechelper.WithContext(ctx),
-		exechelper.WithDir(s.test.ExecutionConfig.PackageRoot))
-	exechelper.WithEnvirons(envs...)
+		exechelper.WithDir(s.test.ExecutionConfig.PackageRoot),
+		exechelper.WithEnvirons(envs...),
```

Comment:
I've locally investigated why tests are failed here. Here is also missed `os.Environ()` (Note: this is already used inside other runners, look at `utils.RunCommand`) 
```suggestion
		exechelper.WithEnvirons(append(os.Environ(), envs...)...),
```

Reply:
Thanks, added.